gcloud-java-bigquery/README.md

```diff
@@ -8,16 +8,22 @@ Java idiomatic client for [Google Cloud BigQuery] (https://cloud.google.com/bigq
 <!-- TODO(mziccard): add in the maven shield once the artifact is pushed to maven -->
 
 -  [Homepage] (https://googlecloudplatform.github.io/gcloud-java/)
-+<!-- TODO(mziccard): add link to API documentatin -->
+
+<!-- TODO(mziccard): add link to API documentation -->
```

Comment:
Any reason not to include it? You already provide one related link bellow `[bigquery documentation][bigquery-api] `